hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.tsx

```diff
@@ -0,0 +1,285 @@
+                                                    <TableCell>
+                                                        {convertToDateTime(
+                                                            pendingOrder.created_at
+                                                        )}
+                                                    </TableCell>
+                                                    <TableCell>
+                                                        <FormControlLabel
+                                                            control={
+                                                                <Checkbox color="primary" />
+                                                            }
+                                                            label=""
+                                                            name={`${row.id}-checkbox`}
+                                                            onChange={
+                                                                props.handleChange
+                                                            }
+                                                            style={{
+                                                                marginLeft: "-15px",
+                                                            }}
+                                                        />
+                                                    </TableCell>
+                                                </TableRow>
+                                            ))}
+                                        </TableBody>
+                                    </Table>
+                                </TableContainer>
+                                <Grid
+                                    container
+                                    justifyContent="flex-end"
+                                    spacing={1}
+                                    style={{ marginTop: "10px" }}
+                                >
+                                    <Grid item>
+                                        <Button
+                                            color="secondary"
+                                            variant="text"
+                                            disableElevation
+                                        >
+                                            Reject Order
+                                        </Button>
+                                    </Grid>
+                                    <Grid item>
+                                        <Button
+                                            color="primary"
+                                            variant="contained"
+                                            type="submit"
+                                            disableElevation
+                                            onClick={() => {
+                                                alert(
```

Comment:
can you move this to an external handleSubmit function and pass it into the Formik component